client/src/App.tsx

```diff
@@ -1,33 +1,22 @@
-import { useState } from "react";
-import reactLogo from "./assets/react.svg";
-import viteLogo from "/vite.svg";
 import "./App.css";
+import ProductCart from "./Components/ProductCard";
```

Comment:
Perhaps, should have a folder name with lower case 'components' in line with other pull requests?